From 364d3af6d9bce771cb2819e1558f55cffdd12544 Mon Sep 17 00:00:00 2001
From: Pekka Paalanen <pekka.paalanen@collabora.com>
Date: Thu, 4 Apr 2019 17:19:54 +0300
Subject: [PATCH 15/45] gl-renderer: does not need matrix.c

The symbols of matrix.c are already exported by libweston, no need to build
them again.

Signed-off-by: Pekka Paalanen <pekka.paalanen@collabora.com>
---
 libweston/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libweston/meson.build b/libweston/meson.build
index aeafcc5d..c93307dc 100644
--- a/libweston/meson.build
+++ b/libweston/meson.build
@@ -431,7 +431,6 @@ if get_option('renderer-gl')
 
 	srcs_renderer_gl = [
 		'gl-renderer.c',
-		'../shared/matrix.c',
 		linux_dmabuf_unstable_v1_protocol_c,
 		linux_dmabuf_unstable_v1_server_protocol_h,
 	]
-- 
2.21.0

